main.py

```diff
@@ -0,0 +1,92 @@
+from fastapi import FastAPI, Request
+import uvicorn
+from cryptography.fernet import Fernet
+from os import popen
+
+app = FastAPI()
+from fastapi.responses import StreamingResponse
```

Comment:
Import first all your stuff. You have an operation in the middle of your imports.